src/components/Main/Main.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './Main.css';
-
+import Animal from '../Animal/Animal';
+import { animals } from '../../data';
+import background from '../../background.png';
 export default function Main() {
-  return <main></main>;
+  return (
+    <main style={{ backgroundImage: `url(${background})` }}>
```

Comment:
nice job with the in-line styling! background looks like it's placed and sized on point!